dwave/system/composites/embedding.py

```diff
@@ -515,6 +516,186 @@ def sample(self, bqm, chain_strength=1.0, chain_break_method=None,
             chain_break_fraction=chain_break_fraction, **parameters)
 
 
+class ParamEmbeddingComposite(dimod.ComposedSampler):
+    """Composite that maps problems to a structured sampler using a
+    parameterized embedding method.
+
+    Args:
+       sampler (dimod.Sampler)
+            Structured dimod sampler.
+
+       embedding_method (object, optional, default=minorminer)
```

Comment:
I am wondering why we wouldn't just pass the `find_embedding` function itself. Even if it was a method on a more complex object, we could still path a reference to the method e.g.
```
class SpecializedEmbeddor:
    def find_embedding(self, S, T, **params):
        ...

embeddor = SpecializedEmbeddor()

sampler = ParamEmbeddingComposite(child_sampler, embedding_method=embeddor.find_embedding)
```